src/installer/Directory.Build.props

```diff
@@ -143,7 +143,7 @@
     <OutputRid Condition="'$(OSGroup)' == 'Windows_NT'">win-$(TargetArchitecture)</OutputRid>
     <OutputRid Condition="'$(OSGroup)' == 'OSX'">osx-$(TargetArchitecture)</OutputRid>
     <OutputRid Condition="'$(OSGroup)' == 'Linux' or '$(OSGroup)' == 'Unix'">linux-$(TargetArchitecture)</OutputRid>
-    <OutputRid Condition="'$(OSGroup)' == 'FreeBSD'">freebsd-$(TargetArchitecture)</OutputRid>
+    <OutputRid Condition="'$(OSGroup)' == 'FreeBSD' or $([MSBuild]::IsOSPlatform('FREEBSD'))">freebsd-$(TargetArchitecture)</OutputRid>
```

Comment:
imo, to reduce the complexity, we should extract al these OS-like and RID-like definitions from:

* src/coreclr/src/System.Private.CoreLib/System.Private.CoreLib.csproj
* src/coreclr/tests/src/Directory.Build.props
* src/installer/Directory.Build.props
* src/libraries/Directory.Build.props
* src/mono/Directory.Build.props

into a single file, something like: `eng/native/build-commons.props`.

ps: it was not easy to track these down for new platforms via huge `./build.sh /v:diag ...` output... (to say the least) 😅

Reply:
Absolutely, we haven't had time to align stuff like this yet. There are some awkward mismatches especially between Unix and Linux.

> ps: it was not easy to track these down for new platforms via huge `./build.sh /v:diag ...` output... (to say the least) 😅

Oof... I'm so sorry. Binlogs (http://msbuildlog.com/) are the way to go. (I've successfully built and used the Avalonia version on macOS if that's stopping you. (I guess I don't know for sure on Linuxy platforms though.))

Reply:
For native builds, os/arch declarations are currently converged in one place `eng/native/build-commons.sh`; was thinking if we could use similar approach for os/rid. I will try to do it. :)

> Binlogs (http://msbuildlog.com/) are the way to go

Nice, thanks! I was trying to find something like that from command-line, then fellback to grep patterns to track down where it matches "previous value = 'freebsd". :D
Maybe command-line (and web) interfaces can be added to the Binlogs utility, in addition to WPF, so we could get [slap](https://github.com/slap-editor/slap/blob/master/README.md)-like experience in cli-only base system installations. 💭